packages/vscode-ide-companion/package.json

```diff
@@ -123,9 +123,11 @@
     "vitest": "^3.2.4"
   },
   "dependencies": {
+    "@google/gemini-cli-core": "file:../core",
```

Comment:
is this dependency needed? it would be good not to depend on core in the IDE companion if we ca.